src/extension.ts

```diff
@@ -1329,14 +1329,17 @@ export async function activate(context: vscode.ExtensionContext) {
   vscode.workspace.registerTextDocumentContentProvider('cmake-tools-schema', new SchemaProvider());
   vscode.commands.executeCommand("setContext", "inCMakeProject", true);
 
-  await vscode.window.withProgress(
+  if (vscode.env.uiKind === vscode.UIKind.Web) {
+    // Do not prompt for users of Web-based Codespaces.
```

Comment:
I think we also want to check for Codespaces on Desktop too.  Talk to @Colengms about what needs to be checked for that. You may not need to do the UIKind check at all...

Reply:
I have changed the way we find codespaces users based on cpp-tools